apps/meteor/app/meteor-accounts-saml/server/lib/SAML.ts

```diff
@@ -430,7 +430,7 @@ export class SAML {
 				};
 
 				await this.storeCredential(credentialToken, loginResult);
-				const url = `${Meteor.absoluteUrl('saml')}/${credentialToken}`;
+				const url = `${Meteor.absoluteUrl('saml')}/${credentialToken}?saml_idp_credentialToken=${credentialToken}`;
```

Comment:
it is never too late:
```suggestion
				const url = Meteor.absoluteUrl(`saml/${credentialToken}?saml_idp_credentialToken=${credentialToken}`);
```